### PR TITLE
Add post on Python wheels

### PR DESCRIPTION
**Pull request by tlestang**

### CODE REVIEW

File: content/tech_post/python_wheels.md [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+    auditwheel repair septembrse-0.0.0-cp38-cp38-linux_x86_64.whl
+
+A new directory `wheelhouse` was created with our manylinux wheel in it.
+
+
+## Runtime dependency on OpenBLAS
+
+There is one <del>detail</del> aspect I glossed over.
+
+Our C++ extension module `ddot` has dynamics dependencies to
```
Comment (iamleeg) [inline content/tech_post/python_wheels.md:381]: should be dynamic

File: content/tech_post/python_wheels.md [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+
+In the current state of things, our friend could install the wheel
+fine, but at the moment they would import the `ddot` extension
+module package things would break: `libopenblas not found`!
+
+The solution to this is rather straightfoward: let's add
+`libopenblas` to the wheel (remember, a wheel is nothing else than
+an archive). This is, however, not enough. We also need to make
+sure that, at runtime, our friend's system (more precisely, the
+dynamics linker) knows where to look to find this shared
```
Comment (iamleeg) [inline content/tech_post/python_wheels.md:413]: dynamic, not dynamics

**Review by iamleeg — APPROVED** [review]
Looks great! I spotted a couple of typos but otherwise good to go.

File: content/tech_post/python_wheels.md [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+-   If there are many dependencies, users have to install them all. Tedious.
+-   This is assuming that, for all these dependencies, there is a package
+    ready in the system's package repository. Otherwise they will have
+    to build the dependency themselves. Tedious, potentially
+    technically challenging, and time consuming.
+-   This is also assuming that a  *compatible version* of the dependency
+    is packaged. This is especially a problem on GNU/Linux, since
+    different distributions package different versions of software. If
+    the extension module for our package was linked against version
+    `x` of a dependency, it's not garanteed to work with version
```
Comment (iamleeg) [inline content/tech_post/python_wheels.md:141]: guaranteed

File: content/tech_post/python_wheels.md (hunk 1 above)
Comment (tlestang) [reply content/tech_post/python_wheels.md:381]: Whoops - conjugating the adjective. My English teachers would be very disappointed :) Thanks Graham!

**Pull request merged**